R/zzz.R

```diff
@@ -0,0 +1,6 @@
+.onAttach <- function(lib, pkg) {
+  if (stats::runif(1) > .8) {
+    msgs <- c("#BlackLivesMatter", "#BlackLivesMatter", "#LGBTQ")
```

Comment:
This is cute, but I wasn't lucky enough to get a message:)

Reply:
i keep pressing CONTROL + SHIFT + L until I get one 😆